game/templates/game/lesson_detail.html

```
@@ -0,0 +1,393 @@
+    {% if lesson.quiz_question %}
+
+    <div class="coming-soon">
+
+        <h3>🎯 Quick Quiz</h3>
+
+        <p>{{ lesson.quiz_question }}</p>
+
+        {% for option in lesson.quiz_options %}
+
+            <button
+                class="quiz-btn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)